### ISSUE TITLE
Allow "Content" to be analyzed

### ISSUE THREAD

**Issue by ssougnez** [DanDiplo/Diplo.LinkChecker#7]
Hi,

The structure of my site is the following:

/nl
/fr
/de
/en

So when I use LinkChecker, I need to do it 4 times (once per language). It's not really an issue but it would be nice to be able to select the real Root node (the content one) to analyze 4 languages in one shot.

Cheers

**Comment by DanDiplo** [comment]
Hi. Thanks, I'll look into this for the next release. Though I'm not sure if the Content Picker built into Umbraco that I use allows you to pick the very "root" item. But if it's possible, I'll see what I can do.

**Comment by ssougnez** [comment]
It might be interesting to have a look at how Umbraco does it for the copy dialog (the "Content" root is present there). If you don't have time, I can have a look if I find time :-)

**Comment by DanDiplo** [comment]
Pull requests are always welcome! :)

Lets see what is possible...

**Comment by ssougnez** [comment]
Ok, I did some research. 

The umbtree directive has some undocumented properties usable to leverage the appearance of the tree. However, one of this property is called "hideheader" but... is not configurable from outside. It is set to "true" by default and the only way to change it is to directly use the directive in the HTML, so it's not possible as you're using `dialogService.contentPicker`.

I'll see with Umbraco HQ if they agree to make this property configurable also via `dialogService.contentPicker`.

**Comment by DanDiplo** [comment]
Yeah, I thought it wasn't possible currently. Let me know if you get any response from them.

**Comment by ssougnez** [comment]
PR created: https://github.com/umbraco/Umbraco-CMS/pull/3629

I guess you'll be able to update Diplo.LinkChecker when 7.13 (or after) is released :-)

I'd like to do it but (in all respect), the architecture of the project makes contributions difficult. I guess that on your machine, the projet "Umbraco7.2.4" contains the files you need but when I clone the repository, it's empty (because in the .gitignore) and I need to recreate the project, modify the sln file, restore the package, etc... Well, not easy :-D

**Comment by DanDiplo** [comment]
Thanks! Yeah, I didn't want to include an entire Umbraco solution in the repo.  But you can just copy the contents of the [App_Plugins](https://github.com/DanDiplo/Diplo.LinkChecker/tree/master/Diplo.LinkChecker/App_Plugins) folder into an existing site (and the DLL into the bin) to test.
